Firebase/Database/Api/FIRDataSnapshot.m

```diff
@@ -87,7 +87,7 @@ - (NSUInteger) childrenCount {
     return [self.node.node numChildren];
 }
 
-- (NSEnumerator *) children {
+- (NSEnumerator<FIRDataSnapshot *> *) children {
```

Comment:
This change needs to be made in a header file in order to be publicly visible to Swift/ObjC users.